mzLib/Quantification/QuantificationWriter.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Omics;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Omics.BioPolymerGroup;
+
+namespace Quantification
+{
+    /// <summary>
+    /// This will need to be implemented or changed or relocated later. Right now it's just a placeholder.
+    /// </summary>
+    public static class QuantificationWriter
```

Comment:
If writing require unique functions, you could possibly do this as an interface and an abstract factory. That way the engine constructor calls a writer factor, passes in quant parameters, and receives either the TMT writer or the LFQ writer.